src/plugins/data_table/data_table.js

```diff
@@ -433,6 +433,16 @@ var LocalDataTable = (function() {
         var title = this.textContent || this.innerText;
         var col = cg.columnConfigByTitle.attributes[title];
 
+        // Try to grab it by attribute, if possible
+        var matches = this.className.match(/(?:^|\s)column-(.*?)(?:\s|$)/);
```

Comment:
You could use column-([^\s]*) to make the regex a little safer. (It will not consume any spaces that way)